src/color.rs

```diff
@@ -243,6 +251,8 @@ impl Colors {
         // INode
         m.insert(Elem::INode { valid: true }, Colour::Fixed(13)); // Pink
         m.insert(Elem::INode { valid: false }, Colour::Fixed(245)); // Grey
+        m.insert(Elem::INodeCount { valid: true }, Colour::Fixed(134));
```

Comment:
It is better to use colors <16. I think we can reuse `13` and `245` here.

Reply:
Also, can we update the colors here to be similar to inode count. Adding colors over 16 might cause issues with the user's terminal colorscheme.